src/main/kotlin/listener/CommandListener.kt

```
@@ -68,6 +71,8 @@ class CommandListener @ListenerConstructor constructor(private val jda: JDA) : L
                 commands[command]!!.handler(event)
             } catch (e: CommandExitException) {
                 event.hook.sendMessage(e.message!!).queue()
+            } catch (e: DatabaseException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)